docs/change_log/index.md

```diff
@@ -7,6 +7,7 @@ Under development: version 3.3.5 (a bug-fix release).
 
 * Make the `slugify_unicode` function not remove diacritical marks (#1118).
 * Fix `[toc]` detection when used with `nl2br` extension (#1160)
+* Automatic Links do not treat missing local-part or domain as a valid email address (#1165).
```

Comment:
I would suggest a small wording change:

```suggestion
* Automatic Links do not treat an email address with missing local part or domain as valid (#1165).
```

Reply:
The rewording suggests that `<@Uxxxx>` was intended to be an email address.  How about:

`* Automatic Links improve email address validation when processing @ inside HTML tags <>.`

Reply:
That's even better.